cpg_pipes/utils.py

```diff
@@ -97,3 +97,18 @@ def can_reuse(
 
     logger.debug(f'Reusing existing {path}. Use --overwrite to overwrite')
     return True
+
+
+def read_secret(project_id: str, secret_name: str) -> str:
+    """
+    Reads the latest version of a GCP Secret Manager secret.
+
+    Unlike cpg_utils.cloud.read_secret, crashes if the secret can not be read.
```

Comment:
Should we make `cpg_utils.cloud.read_secret` more configurable instead in terms of error handling, to avoid this duplication?

Reply:
I'll do that!

Reply:
https://github.com/populationgenomics/cpg-utils/pull/52 @lgruen